src/js/constants/ServiceStatusTypes.js

```diff
@@ -1,7 +1,8 @@
 var ServiceStatusTypes = {
   RUNNING: 0,
   DEPLOYING: 1,
-  SUSPENDED: 2
+  SUSPENDED: 2,
+  NA: 4
```

Comment:
Yeah, but it seems to me that we don't use the key for any sorting here so I guess we're good.